hip_data_tools/apache/cassandra.py

```diff
@@ -0,0 +1,163 @@
+"""
+Utility for connecting to and transforming data in Cassandra clusters
+"""
+import pandas as pd
+from cassandra import ConsistencyLevel
+from cassandra.cluster import ResultSet
+from cassandra.cqlengine.management import sync_table
+from cassandra.query import dict_factory, BatchStatement
+from pandas import DataFrame
+from pandas._libs.tslibs.nattype import NaT
+from pandas._libs.tslibs.timestamps import Timestamp
+
+from hip_data_tools.connect.cassandra import CassandraConnectionManager
+
+
+def _pandas_factory(colnames, rows):
+    return pd.DataFrame(rows, columns=colnames)
+
+
+class CassandraUtil:
+    """
+    Class to connect to and then retrieve, transform and upload data from and to cassandra
+    """
+
+    def __init__(self, keyspace: str, conn: CassandraConnectionManager):
+        self.keyspace = keyspace
+        self._conn = conn
+        self._session = self._conn.get_session(self.keyspace)
+
+    @staticmethod
```

Comment:
should these static methods be extracted to module level? May be easier to test and maintain

Reply:
funny you should say that